src/site/content/en/handbook/content-types/example-post/index.md

```diff
@@ -1,5 +1,6 @@
 ---
 layout: post
+permalink: /handbook/content-types/example-post
```

Comment:
Isn't that the default permalink which is inflected from the file path? Why does it need to be specifically defined here?

Reply:
So, from memory, there's some logic that flattens the URLs of posts. Without that permalink it ends up in the root directory of dist. I thought it'd be better kept within the content-types path for consistency

Reply:
Flatting only happens for `/blog/` (I think 😅). Tried removing it and the page is still available at http://localhost:8080/handbook/content-types/example-post, if that also works for you I'd vote to remove it.

Reply:
Huh - possibly something windows related but just tested again and got same result:
![image](https://user-images.githubusercontent.com/1577555/213215234-f44a19a2-856e-4f4a-b7dd-b1bf3865fdae.png)